packages/hasura-auth-js/src/hasura-auth-client.ts

```diff
@@ -347,6 +358,17 @@ export class HasuraAuthClient {
     throw Error(`Unknown deanonymization method`)
   }
 
+  /**
+   * Use `nhost.auth.addSecurityKey to add a security key to the user, using the WebAuthn API.
+   * @param nickname optional human-readable nickname for the security key
+   *
+   * @docs https://docs.nhost.io/reference/javascript/auth/add-security-key
+   */
+  async addSecurityKey(nickname?: string): Promise<{ error: ErrorPayload | null }> {
+    const { error } = await addSecurityKeyPromise(this._client, nickname)
+    return { error }
+  }
+
```

Comment:
Is there no `removeSecurityKey` method on `nhost.auth`?

Reply:
as explained in the opening of this PR, there is no list/remove in `nhost.auth` as `nhost.auth` has no access to the GraphQL api - `nhost.auth` is in `@nhost/hasura-auth-js`, where as `nhost.graphql` is in `@nhost/nhost-js`. `nhost-js` depends on `hasura-auth-js` and not the other way around. To do so would create a circular dependency

Reply:
Does that mean we only have support for `add` in `@nhost/nhost-js`?

But we have support for `add`, `list`, and `remove` in `@nhost/react`?

Reply:
yes

Reply:
See this comment: https://github.com/nhost/nhost/pull/910#discussion_r968458359